lib/sproxyd.js

```diff
@@ -312,13 +361,16 @@ class SproxydClient {
                     });
                 }
             });
+            return;
```

Comment:
Why the `return` here?

Reply:
forgotten to remove this in my attempt to move the else block one tab left...decided to forgo that change to reduce lines

Reply:
will fix